AjobK/Athena
# 201 login check

closes #201 

 **What is it supposed to do**
- Check if user is logged in on reload of page
- User should be able to logout
- User should see ownership view on profile (Currently able to see based on 'plus square' on their own profile)

 **How can we test your branch**
- Check if user is logged in on reload of page
- Check if user is able to logout
- Check if user is able to see profile ownership


 **What broweser have been checked**
